{"version":1,"ops":[{"type":1,"author":{"id":"6a99c17641340d1f02f70e5180366def2607e2e6"},"timestamp":1581081896,"metadata":{"github-id":"MDU6SXNzdWU1NjE2MzU1ODA=","github-url":"https://github.com/BenLubar/df-ai/issues/70","origin":"github"},"title":"Not sure what's going on.","message":"[df-ai-debug.log](https://github.com/BenLubar/df-ai/files/4170988/df-ai-debug.log)\n\nSeems to be  crashing with this after extended play times, normally I'm not around when it does.","files":null}]}